openquake/commonlib/reportwriter.py

```diff
@@ -93,9 +100,14 @@ def build_report(job_ini, output_dir=None):
     oq = readinput.get_oqparam(job_ini)
     output_dir = output_dir or os.path.dirname(job_ini)
     calc = base.calculators(oq)
+    calc.SourceManager = source.DummySourceManager
+    calc.is_effective_trt_model = lambda result_dict, trt_model: True
```

Comment:
Please add a comment about the fact we are not actually running the calculation, or use a dummy calculator to make it cleaner